delfin/drivers/ibm/storwize_svc/ssh_handler.py

```diff
@@ -0,0 +1,409 @@
+        try:
+            with self.ssh_pool.item() as ssh:
+                self.do_exec(SSHHandler.login_command, ssh)
+        except Exception as e:
+            LOG.error("Failed to login ibm storwize_svc %s" %
+                      (six.text_type(e)))
+            raise e
+
+    def do_exec(self, command_str, ssh):
+        """Execute command"""
+        try:
+            utils.check_ssh_injection(command_str)
+            if command_str is not None and ssh is not None:
+                stdin, stdout, stderr = ssh.exec_command(command_str)
+                res, err = stdout.read(), stderr.read()
+                re = res if res else err
+                result = re.decode()
+        except paramiko.AuthenticationException as ae:
+            LOG.error('doexec Authentication error:{}'.format(ae))
+            raise exception.InvalidUsernameOrPassword()
+        except Exception as e:
+            LOG.error('doexec InvalidUsernameOrPassword error:{}'.format(e))
+            if 'timed out' in str(e):
+                raise exception.SSHConnectTimeout()
+            elif 'No authentication methods available' in str(e) \
+                    or 'Authentication failed' in str(e):
+                raise exception.SSHInvalidUsernameOrPassword()
+            elif 'not a valid RSA private key file' in str(e):
+                raise exception.InvalidPrivateKey()
+            elif 'not found in known_hosts' in str(e):
+                raise exception.SSHNotFoundKnownHosts(self.ssh_pool.ssh_host)
+            else:
+                raise exception.SSHException()
+        return result
+
+    def handle_detail_ssh(self, context, command, method):
+        try:
+            with self.ssh_pool.item() as ssh:
+                ssh_info = self.do_exec(command, ssh)
+            return ssh_info
+        except Exception as e:
+            msg = "Failed to ssh ibm storwize_svc %s: %s" % \
+                  (method, six.text_type(e))
+            raise exception.SSHException(msg)
+
+    def handle_capacity(self, value):
+        capacity = 0
+        if value:
+            if value.isdigit():
+                capacity = float(value)
+            else:
+                unit = value[-2:]
+                capacity = float(value[:-2]) * int(
+                    utils.change_capacity_to_bytes(unit))
+        return capacity
+
+    def get_storage(self, context):
+        try:
+            system_info = self.handle_detail_ssh(context,
+                                                 self.systeminfo_command,
+                                                 'storage')
+            enclosure_info = self.handle_detail_ssh(context,
+                                                    self.enclosure_command,
+                                                    'enclosure')
+            enclosure_res = enclosure_info.split('\n')
+            enclosure = enclosure_res[1].split(':')
+            serial_number = enclosure[7]
+            map = {}
```

Comment:
`map` is more like a variable type, not a variable name, please use a meaningful one of this dict instead of `map`.
Otherwise when someone else read this code, they can not know what's in this `map`.
Same as other places.